src/makerom/rom_header.s

```diff
@@ -12,7 +12,18 @@
 .word  0x00000000               /* Unknown */
 .ascii "THE LEGEND OF ZELDA "   /* Internal ROM name */
 .word  0x00000000               /* Unknown */
+
+#ifdef CONSOLE_WIIVC
+.word  0x00000043               /* Cartridge */
+#else
 .word  0x0000004E               /* Cartridge */
+#endif
+
 .ascii "ZL"                     /* Cartridge ID */
 .ascii "E"                      /* Region */
+
+#ifdef CONSOLE_WIIVC
+.byte  0x00                     /* Version */
+#else
 .byte  0x0F                     /* Version */
+#endif
```

Comment:
we need to find a way to force make recompiling this file when we execute ``make wad`` to make sure people won't get define issues here (it's an important change to make the thing work properly)

though I guess it's fine for now

Reply:
should be fine for now